p2-feature/org.eclipse.collections.repository/category.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
-   <feature url="features/org.eclipse.collections.feature.source_10.1.0.qualifier.jar" id="org.eclipse.collections.feature.source" version="10.1.0.qualifier">
-      <category name="org.eclipse.collections.source"/>
+   <feature url="features/org.eclipse.collections.feature_10.3.0.qualifier.jar" id="org.eclipse.collections.feature" version="10.3.0.qualifier">
```

Comment:
@nikhilnanivadekar did we miss this during 10.2.0?

Reply:
yes possibly 😞 . I wish there was an automated way to do this during every release.